securemeet.py

```
@@ -0,0 +1,907 @@
+        self.audio_buffer: List[np.ndarray] = []
+
+        self.processing_thread = None
+        self.stop_processing = threading.Event()
+
+        self.setup_ui()
+        self.setup_keybindings()
+        self.check_retention_policy()
+
+    # UI -------------------------------------------------------------------------
+    def setup_ui(self):
+        menubar = tk.Menu(self.root); self.root.config(menu=menubar)
```

Ruff (0.12.2)

Multiple statements on one line (semicolon)

(E702)